discoveryApiResearchNowMapping.md

```diff
@@ -0,0 +1,478 @@
+  - Internal record type to return with the work. Either instances or editions.
+- `page` integer (0 indexed)
+- `per_page` integer
+- `sort` array of objects
+  - Objects are formatted as `{"field": field, "dir": dir}`
+- `language` array of languages
+- `years`
+  - This should be formatted as `{"start": year, "end": year}`.
+
+For the DRBB/SCC integration, the DRBB data is fetched using a POST request with a body
+containing `page`, `per_page`, `filters` (years, languages), and `queries` (author/contributor, subject).
+
+## Example Translation
+
+A GET request to the Discovery API endpoint detailed above with this query
+
+```
+?q=hello&filters[subjectLiteral][0]=United%20States&sort=title&sort_direction=asc&search_scope=title&page=1&filters[language][0]=lang%3Aeng&filters[language][1]=lang%3Ager&filters[dateAfter]=1993&filters[dateBefore]=2020
+```
+
+would translate to a POST request with this body for ResearchNow's search endpoint:
+
+```json
+{
+  "queries": [
+    {
+      "field": "title",
+      "query": "hello"
+    },
+    {
+      "field": "subject",
+      "query": "United States"
+    }
+  ],
+  "page": 0,
+  "sort": [
+    {
+      "field": "title",
+      "dir": "asc"
+    }
+  ],
+  "filters": [
+    {
+      "field": "language",
+      "value": "eng"
+    },
+    {
+      "field": "language",
+      "value": "ger"
+    },
+    {
+      "field": "years",
+      "value": {
+        "start": 1993,
+        "end": 2020
+      }
+    }
+  ]
+}
+```
+
+<!-- <style>
+  table ul {
+    list-style: none;
+  }
+</style> -->
+
+## API Parameters Comparison
+
+This table lists, in the first column, the frontend features related to searching and search filters available from the
+frontend. The 2nd and 3rd column list out the Discovery API and ResearchNow API parameters that correlate to that
+frontend functionality.
+
+Single quotes (') are used for frontend terminology. Italics are used to describe frontend features. Code styling (
+e.g. `filters`) is used for the parameters the respective APIs except.
+
+| Research Catalog front end                    | Discovery API                                         | ResearchNow API                                 |
+|-----------------------------------------------| ----------------------------------------------------- | ----------------------------------------------- |
+| _Search field_                                | `q`                                                   | `queries`, "value"                              |
+| _Search field dropdown_                       | `search_scope`                                        | `queries`, "field"                              |
+| <ul>'All Fields'                              | <ul>"all"                                             | <ul>"keyword"                                   |
+| <ul>'Title'                                   | <ul>"title"                                           | <ul>"title"                                     |
```

Comment:
These `ul`s can be better formatted to be `li`s and have closing tags but this is not a deal breaker to merge this PR in right now.